apps/web-giddh/src/app/material-ledger/components/update-ledger-entry-panel/update-ledger-entry-panel.component.html

```diff
@@ -1541,6 +1552,28 @@ <h3 *ngIf="!openDropDown" (dblclick)="openHeaderDropDown()" class="d-inline-bloc
 
                                     <table>
                                         <tr>
+                                            <td
+                                                *ngIf="
+                                                    voucherApiVersion === 2 &&
+                                                    (vm.selectedLedger.voucher?.shortCode === 'credit note' ||
+                                                        vm.selectedLedger.voucher?.shortCode === 'debit note')
+                                                "
+                                                class="pd-r1 pb-1"
+                                            >
+                                                <label class="default-color">{{ commonLocaleData?.app_number }}</label>
+                                                <input
+                                                    matInput
+                                                    id="update-ledger-invoice-number"
+                                                    type="text"
+                                                    [placeholder]="
+                                                        voucherApiVersion === 2
```

Comment:
This check is not required as the parent `td` will only be rendered when `voucherApiVersion` has value `2`